### PR TITLE
Feat: Implements support for Pyro's remote imports

### LINKED ISSUE #195: Git (URL) Imports from .ppj have no support
**Describe the bug**
Repository imports from a `.ppj` aren't recognized as imports. Their missing symbols coat whole files in red, and they don't appear in the Project Explorer.

**To Reproduce**

1. Add an `Import` to your `.ppj` with any repository (e.g. [skse](https://github.com/MrOctopus/nl_online/tree/main/skse64/source))
2. Wait for the Project Explorer window to refresh.

### PR DESCRIPTION
**Pull request by XxX-Daniil-underscore-Zaikin-XxX**
The documentation of Pyro's original feature can be found [here](https://wiki.fireundubh.com/pyro/remotes).

This PR will let the Language Server and Extension find and import the files22. None of the files will be found until Pyro gets a repository key and downloads them, but hopefully, the users will figure that out. I've included tests for much of this functionality. Will close #195.

Additionally, the Project Explorer now includes the name of each import:
image

I've also adjusted various settings to get this code to compile locally and build through GitHub Actions.

### CODE REVIEW

**Comment by XxX-Daniil-underscore-Zaikin-XxX** [comment]
whoops, I realise now the VSCE token was, in fact, important. I'm converting this to draft till I make my own token, put it back in, and test.

**Comment by XxX-Daniil-underscore-Zaikin-XxX** [comment]
ready for review now

File: src/DarkId.Papyrus.LanguageService/Projects/ProgramExtensions.cs [hunk 1]
```diff
@@ -16,14 +17,24 @@ public static ProgramOptionsBuilder WithProject(this ProgramOptionsBuilder build
...
+                    : new SourceInclude()
+                    {
+                        IsImport = true,
+                        Path = Path.GetFullPath(Path.Combine(projectFileDirectory, PathUtilities.Normalize(import)))
+                    })
+                );
 
             if (projectInfo.Project.Folders != null && projectInfo.Project.Folders.Length > 0)
             {
+                // Huh??? does this just... include one folder? and that's it? huh????
```
Comment (joelday) [inline src/DarkId.Papyrus.LanguageService/Projects/ProgramExtensions.cs:37]: Yeah, totally don't remember.

**Review by joelday — APPROVED** [review]
Phenomenally great work. Thank you.

File: src/DarkId.Papyrus.LanguageService/Projects/ProgramExtensions.cs (hunk 1 above)
Comment (XxX-Daniil-underscore-Zaikin-XxX) [reply src/DarkId.Papyrus.LanguageService/Projects/ProgramExtensions.cs:37]: haha I planned to fix this too but forgot

**Pull request merged**